config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
i think as long as it's only a few minutes of extra effort we should make the query param not case sensitive, so `?app=caseflow` or `?app=Caseflow` would work. only the latter would work here. what do you think?